core/database_api.php

```diff
@@ -1323,3 +1323,23 @@ function db_mysql_fix_utf8( $p_string ) {
 function db_empty_result() {
 	return new ADORecordSet_empty();
 }
+
+/**
+ * Returns the maximum value of a valid integer for current driver, for the integer
+ * type that is actually used by Adodb for translating "I UNSIGNED", which is
+ * the type used in mantis schema for most numeric id fields.
+ * Adodb maps "I" as "I4". The actual types used for each driver are mapped in
+ * ADODB_DataDict objects, function ActualType.
+ * @return int
+ */
+function db_max_uint() {
+	# Safe assumption is for maximum value of 4 bytes signed integer: 2^31-1
+	# This is the limit for databases that dont support unsigned types, eg pgsl, mssql
+	$t_max = 2147483647;
+
+	if( db_is_mysql() ) {
```

Comment:
The reasoning for calling it db_max_uint() was:
The biggest number we can use for the type that WE have defined as "unsigned int"
What whe declare as "I UNSIGNED" is mapped to the closest type existing in database. In the case of pgsql, this is a "4 byte signed int"

Anyway, as per the suggestions, the function will be converted to a constant, which is good enough for our purpose. Calling it still DB_MAX_UINT?